react-components/src/analysis/view/AnalysesMenuItems.js

```diff
@@ -60,9 +62,22 @@ class AnalysesMenuItems extends Component {
                     className={classes.menuItem}
                     data-disabled={disableSingleSelectionMenuItem}
                 >
-                    <FolderIcon style={{ color: palette.darkBlue }} />
+                    <FolderIcon lassName={classes.toolbarItemColor} />
                     {getMessage("goOutputFolder")}
                 </MenuItem>
+                <MenuItem
+                    id={build(baseDebugId, ids.MENUITEM_VIEW_LOGS)}
+                    disabled={disableSingleSelectionMenuItem || disableCancel}
+                    onClick={() => {
+                        handleClose();
+                        handleViewLogs();
+                    }}
+                    className={classes.menuItem}
+                    data-disabled={disableSingleSelectionMenuItem}
```

Comment:
Yes. Good point! Just an FYI -  by using `data-<attribute>` lets you pass custom attributes to DOM. This can used in Robotframeworks test to identify if an element is enabled or disabled. 
See https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes

![image](https://user-images.githubusercontent.com/1250790/64290271-d11aff80-cf33-11e9-8aab-13630ba0fa8d.png)